### PR TITLE
Add Helm Chart

### PR DESCRIPTION
**Pull request by skatsaounis**
This PR adds a Helm Chart for Kubernetes installation of Ansible Inventory Server

### CODE REVIEW

File: helm/ansible-inventory-server/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+
+The command deploys Ansible Inventory Server on the Kubernetes cluster in the default configuration. The [Parameters](#parameters) section lists the parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```console
+$ helm delete my-release
```
Comment (neoaggelos) [inline helm/ansible-inventory-server/README.md]: `delete` is just an alias, let's use the canonical name (`uninstall`) instead.

File: helm/ansible-inventory-server/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+The command removes all the Kubernetes components associated with the chart and deletes the release.
+
+## Parameters
+
+The following tables lists the configurable parameters of the Ansible Inventory Server chart and their default values.
+
+| Variable                                  | Description                                 | Default                                                  |
+| ----------------------------------------- | ------------------------------------------- | -------------------------------------------------------- |
+| `ansibleInventoryServerImage.repository`  | Ansible Inventory Server Image name         | `cloudeng/ansible-inventory-server`                      |
```
Comment (neoaggelos) [inline helm/ansible-inventory-server/README.md]: Why not `image.XXXX` ?

Comment (skatsaounis) [reply helm/ansible-inventory-server/README.md]: The intention is to server a public Docker image for anyone interested in using it: https://hub.docker.com/repository/docker/cloudeng/ansible-inventory-server

Comment (skatsaounis) [reply helm/ansible-inventory-server/README.md]: variable name has been changed

**Review by neoaggelos — APPROVED** [review]

**Pull request merged**